composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "simivar/riot-php",
+    "name": "domenikus/riot-php",
```

Comment:
This is the name under which the package is published by Composer on Packagist. This shouldn't be changed as it would break installations.